Building-ChakraCore.md

```diff
@@ -113,3 +113,17 @@ Some details on build script arguments used above:
 * ```--icu=..``` specify the location of ICU library (use `--no-icu` if you don't need unicode support)
 * ```--test-build``` enable command line options and some optimizations
 * ```-j=2``` compile with 2 threads
+
+### Building and debugging in XCode ###
```

Comment:
I don't think this is debugging, maybe testing to be exact?

Reply:
For me the key point of using XCode is that it makes it easy to debug the code.

If you prefer I can change "debugging" to "testing", or maybe just say "Building in XCode".

Reply:
The reason I'm avoiding debugging is to not give people the impression that we have plug-n-go support for script debugging in XCode just like [debugging scripts for Chakra in VS](https://docs.microsoft.com/en-us/scripting/chakra-hosting/hosting-the-javascript-runtime). Either 'testing' or just 'Building in XCode' is fine by me :).

Reply:
Debugging c++ ?

Reply:
I've changed it to just say "Building in XCode".